x-pack/plugins/apm/server/lib/service_map/get_service_map_from_trace_ids.test.ts

```diff
@@ -42,7 +43,13 @@ describe('getConnections', () => {
           (conn) => conn.source['service.environment'] === environment
         )
       ).toBeTruthy();
-      expect(connections).toMatchSnapshot();
+      const serviceNames = new Set();
+      connections.forEach((conn) =>
+        serviceNames.add(conn.source['service.name'])
+      );
+      ['opbeans-python', 'opbeans-node', 'opbeans-ruby'].forEach((name) =>
+        expect(serviceNames.has(name)).toBeTruthy()
+      );
     });
```

Comment:
I don't understand this. The test says to "excludes connections with different service name and environment". Shouldn't `opbeans-python` and `opbeans-ruby` be excluded then?